docs/src/index.md

```diff
@@ -23,6 +23,7 @@ pmt
 rate
 irr
 mirr
+reff
```

Comment:
While the function names are a little terse, the convention has been abbreviations of the specific calculation.  e.g. mirr = modified internal rate of return.  For effective interest rate, I would name the function as `eir`.

Reply:
Sure!